.mk/development.mk

```diff
@@ -22,6 +22,21 @@ undeploy-loki: ## Undeploy loki.
 	curl -S -L https://raw.githubusercontent.com/netobserv/documents/main/examples/zero-click-loki/1-storage.yaml | kubectl --ignore-not-found=true  delete -f - || true
 	-pkill --oldest --full "3100:3100"
 
+.PHONY: deploy-kafka
+deploy-kafka:
+	@echo -e "\n==> Deploy kafka"
+	kubectl create namespace $(NAMESPACE)  --dry-run=client -o yaml | kubectl apply -f -
+	kubectl create -f "https://strimzi.io/install/latest?namespace="$(NAMESPACE) -n $(NAMESPACE)
+	kubectl create -f "https://raw.githubusercontent.com/netobserv/documents/main/examples/kafka-cluster.yaml" -n $(NAMESPACE)
+	kubectl create -f "https://raw.githubusercontent.com/netobserv/documents/main/examples/kafka-topic.yaml" -n $(NAMESPACE)
+
+.PHONY: undeploy-kafka
+undeploy-kafka: ## Undeploy loki.
```

Comment:
change comment from `Undeploy loki` to `undeploy-kafka`

Reply:
Done, thanks!